getting-started/templates/systemlink-values.yaml

```diff
@@ -591,22 +591,22 @@ dashboardhost:
 
 ## Grafana provisioning
 ##
-grafanaprovisioning:
-  grafanaDashboardsProvisioning:
+# grafanaprovisioning:
+  # grafanaDashboardsProvisioning:
     ## Dashboards listed are enumerated over the dashboard-configmap-template.yaml
     ## Each key under the "dashboards" item will be the name of a ConfigMap.
     ## It is recommended to add the "-dashboard" suffix to each key to avoid collision with other ConfigMaps
     ## Dashboard json files can be placed under "dashboards/[project-name]/[dashboard-file].json"
     ## By default, all "enabled" parameters are considered as true even if they are not mentioned. You can disable this by assigning "whitelistMode: true"
     ##
-    whitelistMode: true
-  grafanaDatasourcesProvisioning:
+    # whitelistMode: true
```

Comment:
@CiprianAnton questioned why we had this config set to `true` in these values, but we default to `false` in the chart itself. @pvallone do you recall why we did this?

Reply:
I spoke with Phil offline. We had this in place to account for some O+ dashboards/datasources that are no longer in the chart defaults. We can safely turn this off.